### PR TITLE
Fix incorrect indentation during line breaks in mapping constructors

### PR DESCRIPTION
**Pull request by IrushiL**
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/36171
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/32408

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by NipunaRanasinghe — DISMISSED** [review]
LGTM

File: misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java [hunk 1]
```diff
@@ -3806,6 +3806,59 @@ protected <T extends Node> SeparatedNodeList<T> formatSeparatedNodeList(Separate
...
+                                                                            int listTrailingWS,
+                                                                            int listTrailingNL) {
+        return formatSeparatedNodeList(nodeList,
+                itemTrailingWS,
+                itemTrailingNL,
+                separatorTrailingWS,
+                separatorTrailingNL,
+                listTrailingWS,
+                listTrailingNL,
+                false);
```
Comment (NipunaRanasinghe) [inline misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java:3859]: Any particular reason to use a different formatting style, compared to L3828?

Comment (IrushiL) [reply misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java]: Nope, that was a mistake. Fixed it.

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Comment by NipunaRanasinghe** [comment]
Removing the stale state as the changes are already merged with 
https://github.com/ballerina-platform/ballerina-lang/pull/38308
https://github.com/ballerina-platform/ballerina-lang/pull/38310
https://github.com/ballerina-platform/ballerina-lang/pull/38307

**Pull request closed without merging**